### PR TITLE
[PEN-1461] Added fixes for URL compression

### PR DESCRIPTION
**Pull request by jgrosspietsch**
## Description
Made Thumbor URLs work (I think).

If tests are broken then that's too bad.

## Jira Ticket
- [PEN-1461](https://arcpublishing.atlassian.net/browse/PEN-1461)

## Acceptance Criteria
Not sure

## Test Steps
RUN IT LOCALLY

In your blocks.json you need to set one of your sites to have the property `shouldCompress` set to `true` and at least one other site where it is set to `false` or is unset. Make sure both of those sites work when loading one of our pages with all the blocks.

## Dependencies or Side Effects
See the blocks.json information in the testing steps.

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x] Confirmed all the test steps above are working
- [x] Confirmed there are no linter errors
- [x] Confirmed this PR has reasonable code coverage
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

### CODE REVIEW

**Comment by JackHowa** [comment]
kind of confused by the errors?

```


 PASS  src/components/icons/CameraIcon.test.tsx
 PASS  src/components/Video/index.test.jsx
 PASS  src/components/ImageMetadata/index.test.tsx
 FAIL  src/components/Image/SourceHandler.test.tsx
  ● image source handler component › returns well-formed source tag if correct dimension image passed in

    expect(received).toBe(expected) // Object.is equality

    Expected: "<source srcSet=\"www.hey.resizer.com/correct-image.jpg=/100x100/www.hey.com/ffdfdf\" media=\"screen and (min-width: 200px)\"/>"
    Received: "<source srcSet=\"www.hey.resizer.com//correct-image.jpg=/100x100/www.hey.com/ffdfdf\" media=\"screen and (min-width: 200px)\"/>"

      19 |     expect(wrapper.find('source').prop('media')).toBe('screen and (min-width: 200px)');
      20 | 
    > 21 |     expect(wrapper.html()).toBe('<source srcSet="www.hey.resizer.com/correct-image.jpg=/100x100/www.hey.com/ffdfdf" media="screen and (min-width: 200px)"/>');
         |                            ^
      22 |   });
      23 | 
      24 |   it('returns a null render if no matching dimension found', () => {

      at Object.it (src/components/Image/SourceHandler.test.tsx:21:28)

  ● image source handler component › when compressedParams is true › returns well-formed source tag if correct dimension image passed in

    expect(received).toBe(expected) // Object.is equality

    Expected: "<source srcSet=\"www.hey.resizer.com/correct-image.jpg=/100x100/www.hey.com/ffdfdf\" media=\"screen and (min-width: 200px)\"/>"
    Received: "<source srcSet=\"www.hey.resizer.com//correct-image.jpg=/100x100/www.hey.com/ffdfdf\" media=\"screen and (min-width: 200px)\"/>"

      53 |       expect(wrapper.find('source').prop('media')).toBe('screen and (min-width: 200px)');
      54 | 
    > 55 |       expect(wrapper.html()).toBe('<source srcSet="www.hey.resizer.com/correct-image.jpg=/100x100/www.hey.com/ffdfdf" media="screen and (min-width: 200px)"/>');
         |                              ^
      56 |     });
      57 |   });
      58 | });

      at Object.it (src/components/Image/SourceHandler.test.tsx:55:30)

 PASS  src/components/icons/EnvelopeOpenIcon.test.tsx
 PASS  src/components/icons/GripVerticalIcon.test.tsx
 PASS  src/components/icons/CogIcon.test.tsx
 PASS  src/components/MetaData/index.test.tsx (6.282s)
 FAIL  src/components/Image/image.test.jsx
  ● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /Users/howardj/sites/engine-theme-sdk/src/components/Image/index.tsx:102
        if (Fusion?.isAdmin) {
                   ^

    SyntaxError: Unexpected token .

      2 | import { shallow } from 'enzyme';
      3 | 
    > 4 | import Image from './index';
        | ^
      5 | 
      6 | describe('image component', () => {
      7 |   const breakpoints = {

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:579:25)
      at Object.<anonymous> (src/components/Image/image.test.jsx:4:1)

 PASS  src/components/icons/ZoomInIcon.test.tsx
 FAIL  src/components/Gallery/index.test.tsx
  ● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /Users/howardj/sites/engine-theme-sdk/src/components/Image/index.tsx:102
        if (Fusion?.isAdmin) {
                   ^

    SyntaxError: Unexpected token .

      30 | import { useSwipeable } from 'react-swipeable';
      31 | import PropTypes from 'prop-types';
    > 32 | import Image from '../Image';
         | ^
      33 | import Lightbox from '../Lightbox/index';
      34 | import ImageMetadata from '../ImageMetadata';
      35 | import useInterval from './setInterval';

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:579:25)
      at Object.<anonymous> (src/components/Gallery/index.tsx:32:1)

 PASS  src/components/icons/GridIcon.test.tsx
 PASS  src/components/icons/EllipsisHorizontalIcon.test.tsx
 PASS  src/components/icons/CalendarIcon.test.tsx
 PASS  src/components/icons/ClosedCaptioningIcon.test.tsx
 PASS  src/components/icons/StarIcon.test.tsx
 PASS  src/components/icons/WhatsAppIcon.test.tsx
 PASS  src/components/icons/LockIcon.test.tsx
 PASS  src/components/icons/PhoneIcon.test.tsx
 PASS  src/components/icons/TwitterIcon.test.tsx
 PASS  src/components/icons/LinkedInIcon.test.tsx
 PASS  src/components/icons/ZoomOutIcon.test.tsx
 PASS  src/components/icons/HeadphonesIcon.test.tsx
 PASS  src/components/icons/ChevronLeftIcon.test.tsx
 PASS  src/components/icons/RssIcon.test.tsx
 PASS  src/components/icons/TwitterAltIcon.test.tsx
 PASS  src/components/icons/HamburgerMenuIcon.test.tsx
 PASS  src/components/icons/ShoppingCartIcon.test.tsx
 PASS  src/components/icons/LinkedInAltIcon.test.tsx
 PASS  src/components/icons/PauseCircleIcon.test.tsx
 PASS  src/components/icons/PlayIcon.test.tsx
 PASS  src/components/icons/ChevronDownIcon.test.tsx
 PASS  src/components/icons/PreviousIcon.test.tsx
 PASS  src/components/icons/PinterestAltIcon.test.tsx
 PASS  src/components/icons/RedditIcon.test.tsx
 PASS  src/components/icons/CommentsIcon.test.tsx
 PASS  src/components/icons/PlusIcon.test.tsx
 PASS  src/components/icons/EllipsisVerticalIcon.test.tsx
 PASS  src/components/icons/PauseIcon.test.tsx
 PASS  src/components/icons/ArrowLeftIcon.test.tsx
 PASS  src/components/icons/TumblrIcon.test.tsx
 PASS  src/components/EventEmitter/index.test.tsx
 PASS  src/components/icons/UnlockIcon.test.tsx
 PASS  src/components/icons/PinterestIcon.test.tsx
 PASS  src/components/icons/SnapchatIcon.test.tsx
 PASS  src/components/icons/PrintIcon.test.tsx
 PASS  src/components/icons/PageIcon.test.tsx
 PASS  src/components/icons/YoutubeIcon.test.tsx
 PASS  src/components/icons/YoutubeAltIcon.test.tsx
 PASS  src/components/icons/ChevronRightIcon.test.tsx
 PASS  src/components/icons/InstagramIcon.test.tsx
 PASS  src/components/icons/UserIcon.test.tsx
 PASS  src/components/icons/FacebookIcon.test.tsx
 PASS  src/components/icons/ChevronUpIcon.test.tsx
 PASS  src/components/icons/ExternalLinkIcon.test.tsx
 PASS  src/components/icons/CheckIcon.test.tsx
 PASS  src/components/icons/CloseIcon.test.tsx
 PASS  src/components/icons/ForwardIcon.test.tsx
 PASS  src/components/icons/FacebookAltIcon.test.tsx
 PASS  src/components/icons/DownloadIcon.test.tsx
 PASS  src/components/icons/MediumIcon.test.tsx
 PASS  src/components/icons/BellIcon.test.tsx
 PASS  src/components/icons/FullscreenIcon.test.tsx
 PASS  src/components/icons/SoundOnIcon.test.tsx
 PASS  src/components/icons/GiftIcon.test.tsx
 PASS  src/components/icons/SoundCloudIcon.test.tsx
 PASS  src/components/icons/EnvelopeIcon.test.tsx
 PASS  src/components/icons/StarHalfIcon.test.tsx
 PASS  src/components/icons/LinkIcon.test.tsx
 PASS  src/components/icons/SoundOffIcon.test.tsx
 PASS  src/components/icons/ArrowDownIcon.test.tsx
 PASS  src/components/icons/BackwardIcon.test.tsx
 PASS  src/components/icons/ChartIcon.test.tsx
 FAIL  src/components/Image/thumbor-image-url.test.ts
  ● thumbor image url function › returns a well-formed url image if all params present as expected

    expect(received).toBe(expected) // Object.is equality

    Expected: "www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf"
    Received: "www.hey.resizer.com//r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf"

      14 |   });
      15 |   it('returns a well-formed url image if all params present as expected', () => {
    > 16 |     expect(buildThumborURL(targetImageKey, '100x100', imageSource, resizerURL)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf');
         |                                                                                 ^
      17 |   });
      18 |   it('returns a well-formed url image if filters have been assumed defaulted (with the cm=t param) to a format of jpg and quality of 70', () => {
      19 |     expect(buildThumborURL(targetImageKeyCompressed, '100x100', imageSource, resizerURL, true)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf');

      at Object.it (src/components/Image/thumbor-image-url.test.ts:16:81)

  ● thumbor image url function › returns a well-formed url image if filters have been assumed defaulted (with the cm=t param) to a format of jpg and quality of 70

    expect(received).toBe(expected) // Object.is equality

    Expected: "www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf"
    Received: "www.hey.resizer.com//r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf"

      17 |   });
      18 |   it('returns a well-formed url image if filters have been assumed defaulted (with the cm=t param) to a format of jpg and quality of 70', () => {
    > 19 |     expect(buildThumborURL(targetImageKeyCompressed, '100x100', imageSource, resizerURL, true)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf');
         |                                                                                                 ^
      20 |   });
      21 |   it('retuns well-formed url with no filters', () => {
      22 |     expect(buildThumborURL(targetImageKeyNoFilters, '100x100', imageSource, resizerURL)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100//www.hey.com/ffdfdf');

      at Object.it (src/components/Image/thumbor-image-url.test.ts:19:97)

  ● thumbor image url function › retuns well-formed url with no filters

    expect(received).toBe(expected) // Object.is equality

    Expected: "www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100//www.hey.com/ffdfdf"
    Received: "www.hey.resizer.com//r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100//www.hey.com/ffdfdf"

      20 |   });
      21 |   it('retuns well-formed url with no filters', () => {
    > 22 |     expect(buildThumborURL(targetImageKeyNoFilters, '100x100', imageSource, resizerURL)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100//www.hey.com/ffdfdf');
         |                                                                                          ^
      23 |   });
      24 | });
      25 | 

      at Object.it (src/components/Image/thumbor-image-url.test.ts:22:90)

 PASS  src/components/icons/EditIcon.test.tsx
 PASS  src/components/icons/ArrowUpIcon.test.tsx
 PASS  src/components/icons/NextIcon.test.tsx
 PASS  src/components/icons/HomeIcon.test.tsx
 PASS  src/components/icons/DeleteIcon.test.tsx
 PASS  src/components/icons/SearchIcon.test.tsx
 PASS  src/components/icons/ShareIcon.test.tsx
 PASS  src/components/icons/ArrowRightIcon.test.tsx
 PASS  src/components/icons/TwitchIcon.test.tsx
 PASS  src/components/ArcLogo/index.test.tsx
 PASS  src/components/icons/PlayCircleIcon.test.tsx

Summary of all failing tests
 FAIL  src/components/Image/SourceHandler.test.tsx
  ● image source handler component › returns well-formed source tag if correct dimension image passed in

    expect(received).toBe(expected) // Object.is equality

    Expected: "<source srcSet=\"www.hey.resizer.com/correct-image.jpg=/100x100/www.hey.com/ffdfdf\" media=\"screen and (min-width: 200px)\"/>"
    Received: "<source srcSet=\"www.hey.resizer.com//correct-image.jpg=/100x100/www.hey.com/ffdfdf\" media=\"screen and (min-width: 200px)\"/>"

      19 |     expect(wrapper.find('source').prop('media')).toBe('screen and (min-width: 200px)');
      20 | 
    > 21 |     expect(wrapper.html()).toBe('<source srcSet="www.hey.resizer.com/correct-image.jpg=/100x100/www.hey.com/ffdfdf" media="screen and (min-width: 200px)"/>');
         |                            ^
      22 |   });
      23 | 
      24 |   it('returns a null render if no matching dimension found', () => {

      at Object.it (src/components/Image/SourceHandler.test.tsx:21:28)

  ● image source handler component › when compressedParams is true › returns well-formed source tag if correct dimension image passed in

    expect(received).toBe(expected) // Object.is equality

    Expected: "<source srcSet=\"www.hey.resizer.com/correct-image.jpg=/100x100/www.hey.com/ffdfdf\" media=\"screen and (min-width: 200px)\"/>"
    Received: "<source srcSet=\"www.hey.resizer.com//correct-image.jpg=/100x100/www.hey.com/ffdfdf\" media=\"screen and (min-width: 200px)\"/>"

      53 |       expect(wrapper.find('source').prop('media')).toBe('screen and (min-width: 200px)');
      54 | 
    > 55 |       expect(wrapper.html()).toBe('<source srcSet="www.hey.resizer.com/correct-image.jpg=/100x100/www.hey.com/ffdfdf" media="screen and (min-width: 200px)"/>');
         |                              ^
      56 |     });
      57 |   });
      58 | });

      at Object.it (src/components/Image/SourceHandler.test.tsx:55:30)

 FAIL  src/components/Image/image.test.jsx
  ● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /Users/howardj/sites/engine-theme-sdk/src/components/Image/index.tsx:102
        if (Fusion?.isAdmin) {
                   ^

    SyntaxError: Unexpected token .

      2 | import { shallow } from 'enzyme';
      3 | 
    > 4 | import Image from './index';
        | ^
      5 | 
      6 | describe('image component', () => {
      7 |   const breakpoints = {

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:579:25)
      at Object.<anonymous> (src/components/Image/image.test.jsx:4:1)

 FAIL  src/components/Gallery/index.test.tsx
  ● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /Users/howardj/sites/engine-theme-sdk/src/components/Image/index.tsx:102
        if (Fusion?.isAdmin) {
                   ^

    SyntaxError: Unexpected token .

      30 | import { useSwipeable } from 'react-swipeable';
      31 | import PropTypes from 'prop-types';
    > 32 | import Image from '../Image';
         | ^
      33 | import Lightbox from '../Lightbox/index';
      34 | import ImageMetadata from '../ImageMetadata';
      35 | import useInterval from './setInterval';

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:579:25)
      at Object.<anonymous> (src/components/Gallery/index.tsx:32:1)

 FAIL  src/components/Image/thumbor-image-url.test.ts
  ● thumbor image url function › returns a well-formed url image if all params present as expected

    expect(received).toBe(expected) // Object.is equality

    Expected: "www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf"
    Received: "www.hey.resizer.com//r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf"

      14 |   });
      15 |   it('returns a well-formed url image if all params present as expected', () => {
    > 16 |     expect(buildThumborURL(targetImageKey, '100x100', imageSource, resizerURL)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf');
         |                                                                                 ^
      17 |   });
      18 |   it('returns a well-formed url image if filters have been assumed defaulted (with the cm=t param) to a format of jpg and quality of 70', () => {
      19 |     expect(buildThumborURL(targetImageKeyCompressed, '100x100', imageSource, resizerURL, true)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf');

      at Object.it (src/components/Image/thumbor-image-url.test.ts:16:81)

  ● thumbor image url function › returns a well-formed url image if filters have been assumed defaulted (with the cm=t param) to a format of jpg and quality of 70

    expect(received).toBe(expected) // Object.is equality

    Expected: "www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf"
    Received: "www.hey.resizer.com//r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf"

      17 |   });
      18 |   it('returns a well-formed url image if filters have been assumed defaulted (with the cm=t param) to a format of jpg and quality of 70', () => {
    > 19 |     expect(buildThumborURL(targetImageKeyCompressed, '100x100', imageSource, resizerURL, true)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf');
         |                                                                                                 ^
      20 |   });
      21 |   it('retuns well-formed url with no filters', () => {
      22 |     expect(buildThumborURL(targetImageKeyNoFilters, '100x100', imageSource, resizerURL)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100//www.hey.com/ffdfdf');

      at Object.it (src/components/Image/thumbor-image-url.test.ts:19:97)

  ● thumbor image url function › retuns well-formed url with no filters

    expect(received).toBe(expected) // Object.is equality

    Expected: "www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100//www.hey.com/ffdfdf"
    Received: "www.hey.resizer.com//r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100//www.hey.com/ffdfdf"

      20 |   });
      21 |   it('retuns well-formed url with no filters', () => {
    > 22 |     expect(buildThumborURL(targetImageKeyNoFilters, '100x100', imageSource, resizerURL)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100//www.hey.com/ffdfdf');
         |                                                                                          ^
      23 |   });
      24 | });
      25 | 

      at Object.it (src/components/Image/thumbor-image-url.test.ts:22:90)


Test Suites: 4 failed, 81 passed, 85 total
Tests:       5 failed, 966 passed, 971 total
Snapshots:   0 total
Time:        14.83s, estimated 15s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
howardj@it-20750 engine-theme-sdk %

**Review by JackHowa — CHANGES_REQUESTED** [review]
looks like tests aren't passing?

**Comment by JackHowa** [comment]
Not sure if this would help with the resizer url test issues https://github.com/WPMedia/engine-theme-sdk/pull/122/files#diff-10447a794f684347e9f8360cc953631acbf5201a16d26e3868c478fb1a58ba6a

File: src/components/Image/index.tsx [hunk 1]
```diff
@@ -123,8 +123,11 @@ const Image: React.FC<ImageProps> = ({
   // if url passed in directly without resized params
   if (typeof resizedImageOptions === 'undefined' || typeof resizedImageOptions[`${largeWidth}x${largeHeight}`] === 'undefined') {
     // todo: remove for prod
-    console.error(`no resized options found for url: ${url}.`);
-    console.error('Please use resized options to save money on serving bigger images than necessary. Consider using resizer content source block or adding the resizer block to content source transform.');
+    if (Fusion?.isAdmin) {
```
Comment (JackHowa) [inline src/components/Image/index.tsx:126]: don't really know what's happening here? can this be a separate pr

Comment (jgrosspietsch) [reply src/components/Image/index.tsx:126]: Well then we both don't know what's going on.
